### DISCUSSION TITLE
CTB Broken PP system with EZ

### DISCUSSION THREAD

**Discussion by DankeRonnenberg** [ppy/osu#24554] [Q&A]
### Type

Game behaviour

### Bug description

First of all my english is terrible but i wanna submit this huge problem at all.

I play osu! since a long time and play ctb since some weeks again and also play some new rankt maps.
But i notice something odd. and i lookt in the top 50 and see how some players played with DT EZ 
just thought there played it for fun but then I saw the pp they got from the DT EZ play.... 335 pp with 100%
i played the map without any mods and got 79 pp and other people played with DT HD and just got 173 pp
But how the heck is that possible? Thats completely broken and need a fix or CTB competitive
becomes to a big joke.... difficulty increase by a lot when i turn on EZ in the select screen.

### Screenshots or videos

![KLJLVKJLXCJVXVBXBKXCJLVXVX](https://github.com/ppy/osu/assets/59347588/6aab3904-3023-4796-bbb8-cbbddabe8e7d)
![screenshot184](https://github.com/ppy/osu/assets/59347588/5bcc7b03-5dcb-46a3-83e7-5e76a052a16f)
![screenshot185](https://github.com/ppy/osu/assets/59347588/0ae6b380-9f3b-431a-ad82-543a9febc55e)

### Version

Stable 20230814.2

### Logs

[osu!auth.log](https://github.com/ppy/osu/files/12343632/osu.auth.log)
[runtime.log](https://github.com/ppy/osu/files/12343633/runtime.log)
[session.log](https://github.com/ppy/osu/files/12343634/session.log)
[update_success.log](https://github.com/ppy/osu/files/12343635/update_success.log)

**Comment by K3VRAL** [comment]
The Easy mod is a lot more difficult to play due to edge jumps is generates by reducing the circle size, as edge jumps are more difficult then hyper jumps. It's misleading that Easy "reduces difficulty", as the mod menu says, but I think that's more of a byproduct of the original implementation.
I'd recommed you try the map with the Easy mod, and see if you can beat just as easily without it.

**Comment by bastoo0** [comment]
This issue comes from the edge dashes/jumps buff implementation in the SR algorithm. It only happens because EZ generates edge jumps.
From my experience, it cannot be fixed unless we want to rework the edge jumps buff, which should probably be one part of a larger rework.
For now, I don't think this issue is really game breaking, it's only limited to some beginners using a few EZ maps to get a few PP, from what I understand.
